contrib/builder/deb/armhf/raspbian-jessie/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM resin/rpi-raspbian:jessie
```

Comment:
Those should be used with GOARM 5 I think but that is one option.

In contrib/builder/deb/armhf/raspbian-jessie/Dockerfile
<https://github.com/docker/docker/pull/21895#discussion_r63225616>:

> @@ -0,0 +1,11 @@
> +FROM resin/rpi-raspbian:jessie

Why not armel/debian:jessie?
That one works with GOARM 6...

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub
<https://github.com/docker/docker/pull/21895/files/fa10270f1fd14958c3a95adc99131408cd700226#r63225616>